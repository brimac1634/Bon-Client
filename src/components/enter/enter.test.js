import { shallow } from "enzyme";
import React from "react";
import Enter from "./enter.component";

it("expect to render Enter component", () => {
  expect(shallow(<Enter />).debug()).toMatchSnapshot();
});
