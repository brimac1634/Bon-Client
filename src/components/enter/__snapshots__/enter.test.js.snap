// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Enter component 1`] = `"<div className=\\"enter null\\" />"`;
